homeassistant/components/huawei_lte/device_tracker.py

```diff
@@ -81,7 +82,7 @@ def async_add_new_entities(hass, router_url, async_add_entities, tracked):
         _LOGGER.debug("%s[%s][%s] not in data", KEY_WLAN_HOST_LIST, "Hosts", "Host")
         return
 
-    new_entities = []
+    new_entities: List[Entity] = []
```

Comment:
`List` is not covariant, so this would cause mypy to barf:
```
homeassistant/components/huawei_lte/device_tracker.py:103: error: Argument 1 has incompatible type "List[HuaweiLteScannerEntity]"; expected "List[Entity]"
homeassistant/components/huawei_lte/device_tracker.py:103: note: "List" is invariant -- see http://mypy.readthedocs.io/en/latest/common_issues.html#variance
homeassistant/components/huawei_lte/device_tracker.py:103: note: Consider using "Sequence" instead, which is covariant
```
(Aside, our function parameter type hints should in general be changed towards abstract classes ("interfaces") as appropriate instead of using concrete types like `List` (concrete return types are fine). Thwould improve this at would be the right thing to do in general, and would improve things in this case as a side effect.)